### DISCUSSION TITLE
Raspberry Pi Pico modchip

### DISCUSSION THREAD

**Discussion by ableto288** [redolution/gekkoboot#6] [General]
(Sorry for bad english) Could be possible port this code to a raspberry pi pico?

**Comment by 9ary** [comment]
And yet here I am. :P The reason why iplboot is abandoned is because it has been difficult for me to maintain while supporting all the configurations I wanted to. Ideally I would need access to a Qoob SX and a Viper of my own so I could support them more properly (I only have a Qoob Pro). The other big hurdle has been dkppc and libogc which will have to go.

This repo only contains a loader application that runs on the GameCube itself. This code can't be used to make an IPL replacement modchip, however it could be loaded on any other IPL replacement you can come up with. BadAss Consoles's HyperBoot is a good example of this: I helped develop the hardware, and iplboot was used as the basis for the loader menu.

It is definitely possible to make one with a pico, and I would like to explore that eventually. I have a few of those in a drawer.

All of this is on the backburner currently, I have other projects (GameCube related and unrelated) to take care of before I can get back to this. Also probably need a job to fund all this cool stuff.

**Comment by 9ary** [comment]
Keep an eye out for webhdx's PicoBoot to be released soon. I'm currently testing it myself.
https://twitter.com/webhdx/status/1538122564299538434
